src/keepers/HarvestResolver.sol

```diff
@@ -31,7 +36,8 @@ contract HarvestResolver is IResolver, Ownable {
   event SetHarvestJob(
     bool active,
     address alchemist,
-    address rewardToken,
+    address aaveToken,
```

Comment:
why do we need to accept "aaveToken"? shouldnt this fall into either "yieldToken" or "rewardToken"?